security/intrusion-detection-content-snort-vrt/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=		intrusion-detection-content-snort-vrt
+PLUGIN_VERSION=		0.1
+PLUGIN_COMMENT=		IDS Snort VRT ruleset (needs registration or subscription)
+PLUGIN_MAINTAINER=	ad@opnsense.org
```

Comment:
I am not sure if we should add ports for other maintainers @AdSchellevis, normally maintainer is submitter

Reply:
@fichtner @shonjir I don't mind in this case, the template and code behind it is mine, but if @shonjir wants to maintain that's fine by me too, just let me know.